tools/extract_latest_log.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+"""Extract the most recent INFO log from a directory."""
+
```

Comment:
Eh... What about `ls *.log -tr | head -n 1` <- most recently modified file? 😆 
Or `ls -a | grep log | sort | head -n 1` for sorting by name like your script

Reply:
I like the first one! Also, we don't need `r` here, since the order is `descending` by default